src/test/steps/common.ts

```diff
@@ -195,10 +194,6 @@ When('I enter my valid case reference and valid access code', async () => {
   iClick('Continue');
 });
 
-When('a case worker issues the application', async () => {
-  await triggerAnEvent(CASEWORKER_ISSUE_APPLICATION, { ceremonyPlace: 'Somewhere' });
-});
-
```

Comment:
Why remove this? It's used in a couple of places still

Reply:
After chatting yesterday, this has been manually added back to get the PR working. We will be investigating why the event was removed in the first place but this PR can still progress.